trappy/plotter/Constraint.py

```diff
@@ -276,6 +276,13 @@ def _expand(self):
                    Len[columns] != 1
                    Len[templates] != 1
             )
+
+            TODO  is the following correct?
+            Permute(
+                   Len[traces] != 1
+                   Len[columns] = 1
```

Comment:
Great, I'll remove the "TODO" and amend. I've figured out what `signals` is (I think it's explained in another function) so perhaps I can add a comment for that too :)